src/how-to-build-and-run.md

```diff
@@ -300,9 +484,10 @@ in other sections:
 
 ### ctags
 
-One of the challenges with rustc is that the RLS can't handle it, since it's a bootstrapping
-compiler. This makes code navigation difficult. One solution is to use `ctags`. The following
-script can be used to set it up: [https://github.com/nikomatsakis/rust-etags][etags].
+One of the challenges with `rustc` is that the RLS can't handle it, since it's a
+bootstrapping compiler. This makes code navigation difficult. One solution is to
+use `ctags`. The following script can be used to set it up:
+[https://github.com/nikomatsakis/rust-etags][etags].
```

Comment:
btw @nikomatsakis do you think this is necessary?. I used to have also a ctags config file for Rust but seems like ctags handles that automatically now. Is this still needed?. Anyway, this is a thing for a different PR.

Reply:
I don't know, did somebody add ctags support for Rust at some point?

Reply:
I don't see it in [this list](http://ctags.sourceforge.net/languages.html), but who knows. That web page is **sourceforge** so I feel like there is no way it could have been updated in the last 2 decades.

Reply:
Yeah, that website seems very old. There's even a link to [Freshmeat](http://freshmeat.net/) :joy:.
My Linux distro uses https://ctags.io/, the source code is located https://github.com/universal-ctags/ctags and from this https://github.com/universal-ctags/ctags/pull/206 seems like support was added in 2015.

Reply:
Anyone who uses exuberant-ctags will be missing the definition. To my knowledge universal-ctags doesn’t  come on any ubuntu package list, so I didn’t know about it until recently when tmandry pointed it out on urlo.

Reply:
@mark-i-m good to know, that explains then. I'm using Archlinux and the ctags version provided is universal-ctags.

Reply:
It would definitely be worth it to open another PR to add info about universal-ctags. It is far superior to exuberant-ctags + config file.

Reply:
https://github.com/rust-lang/rustc-guide/pull/314